src/component/UI/error/Error.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classes from './Error.module.css';
+interface ErrorMessageProps {
+  message: string;
+}
+
+export class ErrorMessage extends React.Component<ErrorMessageProps> {
+  constructor(props: ErrorMessageProps) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <>
+        <div key={Date.now()} className={classes.error}>
```

Comment:
You should not use key in the case. Moreover Date.now() will provide new value each rerender